README.md

```diff
@@ -242,7 +242,7 @@ The project also includes many example programs and tools using the `llama` libr
 
 - Clone this repository and build locally, see [how to build](docs/build.md)
 - On MacOS or Linux, install `llama.cpp` via [brew, flox or nix](docs/install.md)
-- Use a Docker image, see [documentation for Docker](docs/docker.md)
+- Use a container image, see [documentation for containers](docs/container.md)
```

Comment:
```suggestion
- Use a container image (docker/podman), see [documentation for containers](docs/container.md)
```

In case someone do a Ctrl+F search for "docker", they should see it. Also good for SEO.